corona-warn-app/cwa-app-android
# Remove static context access from databinding accessibility formaters (EXPOSUREAPP-2746)

Small steps, small PRs, I'm trying :heart: 
This PR is part of the viewmodel refactoring series to enable automated screen testing and screenshots.

This specific PR replaces inline XML code that calls a formatter with "BindingAdapters".
It's really similar, but the big win here is that the new approach no longer uses inline XML code, and no longer makes use of static access to the application context  (which breaks testing/mocking)

## How to test
* Review the code
* Find a few of the changed layouts, enable TalkBalk, and check that the "contentDescription" is still being correctly read (i.e. Homescreen Title icon should say "Corona-WarnApp Image")
